hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -285,7 +300,15 @@ public boolean setProperties(Namespace namespace,  Map<String, String> propertie
     parameter.putAll(properties);
     Database database = convertToDatabase(namespace, parameter);
 
-    return alterHiveDataBase(namespace, database);
+    boolean result = alterHiveDataBase(namespace, database);
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("Set properties for: {} to: {} resulted: {}",
+          namespace,
+          Joiner.on(",").withKeyValueSeparator("=").join(properties),
+          result);
```

Comment:
I'm really skeptical that this is a good idea. Properties can contain sensitive information that we don't want to leak into logs. If the operation succeeds, we can get the properties directly from the table. If it fails, we will get an exception and the properties that were attempted aren't very important (the user should also be able to tell us what they were).

Reply:
Good point!
Removed the printing of the parameters.